doc/rtd/development/datasource_creation.rst

```diff
@@ -118,33 +138,19 @@ Add tests for datasource module
 -------------------------------
 
 Add a new file with some tests for the module to
-:file:`cloudinit/sources/test_<yourplatform>.py`. For example, see
-:file:`cloudinit/sources/tests/test_oracle.py`
-
-Update ``ds-identify``
-----------------------
-
-In ``systemd`` systems, ``ds-identify`` is used to detect which datasource
```

Comment:
Added mention of Alpine and BSD too as this is broader than just systemd nowadays